Deck.py

```diff
@@ -0,0 +1,145 @@
+                               "matters.", "Accept the consequences of your decisions. Go through it, get it over "
+                               "with, and free yourself up for new pursuits.", "Free yourself from a past that no "
+                               "longer serves you well and proceed toward the future.",
+                               "Identify and acquire the ingredients that will most help you complete your mission and "
+                               "leave the rest behind.","Let go of inhibitions. Allow yourself to express all of who "
+                               "you are.", "You are the one to serve as a catalyst for change.",
+                               "Your time is better spent in reflection and spiritual pursuit.",
+                               "Listen to the body and its unique wisdom. Rely upon your inner resources as your best "
+                               "source of support and security.", "Let your light shine. Be confident in the sacred "
+                               "power of your original nature.", "Let go of your past. The future"
+                               " welcomes you with a bounty of growth and change.", "Be serene in knowing you are "
+                               "succeeding in your goal."]
+        # add all the cards in the major arcana
+        while self.i < 22:
+            self.deck.append(TarotCard(0, self.i, self.major_arcana[self.i], self.major_meanings[self.i]))
+            self.i = self.i + 1
+
+        # add all the cards in the minor arcana
+        # wands
+        self.i = 0
+        while self.i < 14:
+            self.deck.append(TarotCard(1, self.i, self.minor_ranks[self.i] + " of " + self.minor_suits[0],
+                                       self.wands_meanings[self.i]))
+            self.i = self.i + 1
+        # cups
+        self.i = 0
+        while self.i < 14:
+            self.deck.append(TarotCard(2, self.i, self.minor_ranks[self.i] + " of " + self.minor_suits[1],
+                                       self.cups_meanings[self.i]))
+            self.i = self.i + 1
+        # swords
+        self.i = 0
+        while self.i < 14:
+            self.deck.append(TarotCard(3, self.i, self.minor_ranks[self.i] + " of " + self.minor_suits[2],
+                                       self.swords_meanings[self.i]))
+            self.i = self.i + 1
+
+        # coins
+        self.i = 0
+        while self.i < 14:
+            self.deck.append(TarotCard(4, self.i, self.minor_ranks[self.i] + " of " + self.minor_suits[3],
+                                       self.coins_meanings[self.i]))
+            self.i = self.i + 1
+
+    def __shuffle__(self):
+        random.shuffle(self.deck)
+
+    def __draw__(self, number):
+        self.__shuffle__()
+        # removes and returns card at number index
+        return self.deck.pop(number)
+
+    def __cardsLeft__(self):

```

Comment:
nitpick: `cards_left(self):`. <-- this is "snack case" instead of "camel case" that you have.  if you're writing python, use snake case. If you're writing JavaScript, use camelCase.